app/controllers/tutorials_controller.rb

```diff
@@ -1,5 +1,6 @@
 class TutorialsController < ApplicationController
   before_action :set_tutorial, only: [:show, :edit, :update, :destroy]
+  before_action :require_login, only: [:new, :update, :destroy]
```

Comment:
Devise has a helper for this, I believe it's `user_authorized`?  Check their docs and use that instead.